src/tools/reverseEvent/reverse-event.ts

```diff
@@ -0,0 +1,73 @@
+import fs from "fs-extra";
+
+type logType = {
+  "_id": string,
+  "rid": string,
+  "command": number,
+  "args": string[]
+}
+
+async function reverseEvent() {
```

Comment:
As we discussed, we want to call reverseEvent(doc), which doc is:
const doc = {
        rid: rid,
        command: op,
        args: args.map(x => x.toString())
      };

in the monitor/src/substrate/handler/eventStorage.ts

Thus this function need have a parameter "doc". You can read the example json in main and pass to the function.

Reply:
I understood. I'll modify it.